tests/qiskit_convert_test.py

```diff
@@ -1013,3 +1013,27 @@ def test_failed_conversion_error() -> None:
         NotImplementedError, match=r"Conversion of qiskit's xx_plus_yy instruction"
     ):
         qiskit_to_tk(qc)
+
+
+def test_RealAmplitudes_numeric_params() -> None:
+    qc = QuantumCircuit(3)
+
+    params = [np.pi / 2] * 9
+    real_amps1 = RealAmplitudes(3, reps=2)
+
+    real_amps2 = real_amps1.assign_parameters(params)
+    qc.compose(real_amps2, qubits=[0, 1, 2], inplace=True)
+
+    converted_tkc = qiskit_to_tk(qc)
+    assert converted_tkc.n_gates == 1
+    assert converted_tkc.n_gates_of_type(OpType.CircBox) == 1
+    circbox_op = converted_tkc.get_commands()[0].op
+    assert circbox_op.get_circuit().name == "RealAmplitudes"
+    DecomposeBoxes().apply(converted_tkc)
+    assert converted_tkc.n_gates_of_type(OpType.CX) == 4
+    assert converted_tkc.n_gates_of_type(OpType.Ry) == 9
+    unitary1 = converted_tkc.get_unitary()
+    qc2 = tk_to_qiskit(converted_tkc)
+    tkc2 = qiskit_to_tk(qc2)
+    unitary2 = tkc2.get_unitary()
+    assert compare_unitaries(unitary1, unitary2)
```

Comment:
For completeness can we also check the unitary against that of the original qiskit circuit?

```python
from qiskit.quantum_info import Operator

unitary0 = Operator(qc).data
```

(Might have to correct for endianness?)

Reply:
Yeah makes sense. Done.